deployment/terraform/container_service.tf

```diff
@@ -51,6 +51,16 @@ resource "aws_security_group" "batch" {
   }
 }
 
+resource "aws_security_group" "opensearch" {
```

Comment:
Since it's not related to ECS containers, could it be moved to the opensearch.tf file?